bundles/org.openhab.binding.bigassfan/README.md

```diff
@@ -9,6 +9,11 @@ There is a single thing created for each fan, light, or controller that's connec
 Each thing has channels that allow control of the fan and light, as well as to monitor the status.
 When a fan or light is controlled from the remote control, Wall Controller, or smartphone app, the openHAB items linked to the device's channels will be updated to reflect the status.
 
+### Important Notice
+
+The latest firmware for Haiku devices no longer uses the SenseMe protocol.
```

Comment:
Does it have a version number? Users may wonder (even months or years from now) exactly which version to avoid, and if possible to upgrade to the latest version before that.

Reply:
Yeah, I'm trying to find the version number.

Reply:
Excellent, thanks.